CHANGELOG.md

```diff
@@ -1,9 +1,100 @@
 # did-veres-one ChangeLog
 
+## 14.0.0 -
+
+### `14.0.0-beta.2` - 2021-08-19
+
+### Added
+- Add `driver.getInitial()` method, to explicitly construct a DID Document
+  deterministically, from a cryptonym DID.
+- Add an optional `seed` param to `generate()` to generate DID document from
+  a 32-byte array seed.
+
```

Comment:
```suggestion
```

Reply:
This is in the 14.x branch.  The module conversion branch, aka 15.x, cleans up the changelog.  Has a fix for this and adds entries for other 14.x releases.  I think we can leave this blank line atrocity here for now.